apps/ckit-app/src/components/OnLoaded/index.tsx

```diff
@@ -1,28 +1,36 @@
 import { UnipassWallet, AbstractTransactionBuilder } from '@ckit/ckit';
+import { Transaction } from '@lay2/pw-core';
 import { Spin } from 'antd';
 import React, { useEffect, useState } from 'react';
 import { WalletContainer } from 'containers';
 import { useSendTransaction } from 'hooks/useSendTransaction';
+import { useUnipassTxStorage } from 'hooks/useUnipassTxStorage';
 
 export const OnIncomplete: React.FC = ({ children }) => {
   const [isLoaded, setIsLoaded] = useState(false);
-  const { currentWallet } = WalletContainer.useContainer();
+  const { isInitialized, setCurrentWalletName } = WalletContainer.useContainer();
+  const [unipassTx] = useUnipassTxStorage();
```

Comment:
so many UniPass-related works, maybe we can refactor to
```ts
interface UseUnipassState {
  shouldLoginViaUnipass: boolean;
  shouldSendIncompleteTx: boolean;
  putTxToCache: () => Promise<void>;
  popTxFromCache: () => Promise<Deserialized>;
}

function useUnipass(): UseUnipassState {
  ...TODO
}
```

Reply:
fixed